src/gen_experiments/__init__.py

```diff
@@ -228,11 +235,11 @@ def _convert_abs_rel_noise(grid_vals: list, grid_params: list, recent_results: d
     "pde": ParamDetails(
         ND({
             "featcls": "pde",
-            "library_functions": [identity],
-            "function_names": [identity],
+            "library_functions": [lambda x: x, lambda x: x * x],
+            "function_names": [lambda x: x, lambda x: x + x],
```

Comment:
`mitosis` does not allow lambda functions in `Parameter` objects.  That was why we switched to `identity`.  Running experiments in this branch hits this error.